docs/hellonode.md

```diff
@@ -103,6 +103,14 @@ Visit your app in the browser, or use `curl` or `wget` if you’d like :
 $ curl http://localhost:8080
 Hello World!
 ```
```

Comment:
Please add a blank line here, after the triple-backtics, just to give some breathing room between the code example and the text. (Sometimes they fight. It can get ugly.)